kaldb/src/test/java/com/slack/kaldb/elasticsearchApi/OpenSearchBulkEndpointTest.java

```diff
@@ -268,6 +269,7 @@ record ->
   }
 
   @Test
+  @Disabled("Flaky test")
```

Comment:
This is intermittently getting stuck in execution, even with a higher timeout. Disabling for now as it's also failing master - https://github.com/slackhq/kaldb/actions/runs/6242013937/job/16945192608